app_builder/lib/app_builder/macos.ex

```diff
@@ -107,6 +107,7 @@ defmodule AppBuilder.MacOS do
     File.mkdir_p!(Path.join([app_bundle_path, "Contents", "Resources"]))
     File.rename!(release.path, Path.join([app_bundle_path, "Contents", "Resources", "rel"]))
 
+    File.mkdir_p!("tmp")
```

Comment:
Should we use `System.tmp_dir`? 🤔

Reply:
Probably. 
@wojtekmach Something speaking against it?

Reply:
`System.tmp_dir` won't survive system reboot. So next time when we run `build_mac.sh`, which runs `bootstrap_otp.sh`, we'd compile wx & OTP again which will be slow. I'd keep it on project's `tmp/` for this reason.

Reply:
Ah, so we should also not remove the `tmp` directory, correct?

Reply:
yes no!

Reply:
@wojtekmach 
Okay, then my `File.rm_rf!("tmp/")` wouldn't make sense either. I just removed it.
Should the "real" temporary files be put into `System.tmp_dir` e.g. `launcher.swift` and the temp dmg or would it make things unnecessarily complicated?